### ISSUE TITLE
Helicopter's explosion bug

### ISSUE THREAD

**Issue by litolax** [altmp/altv-issues#1638]
### Description of the problem

When I open client first time and connect to the server with helicopters, after I come to them I hear explosion sound and one of them is destroyed 

### Reproduction steps

1) Open client first time
2) Start server 
```csharp
public override void OnStart()
    {
        Alt.OnPlayerConnect += (player, reason) =>
        {
            player.Spawn(new Position(100,100,80));
            Alt.CreateVehicle(VehicleModel.T20, new Position(100,100,80), player.Rotation);
            player.Model = Alt.Hash("mp_m_freemode_01");
        };
        
        
        Alt.CreateVehicle(VehicleModel.Frogger2,
            new Position(122.25234752597993f, -739.2074051069882f, 262.9959716796875f),
            new Rotation(0, 0, 1.9693796634674072f));
        Alt.CreateVehicle(VehicleModel.Frogger2,
            new Position(147.28561974491578f, -755.8065605442248f, 263.05157470703125f),
            new Rotation(0, 0, 2.873450756072998f));

        
        Alt.CreateVehicle(VehicleModel.Polmav,
            new Position(448.322998046875f, -981.1299438476562f, 44.178609466552734f),
            new Rotation(0.0022603238467127085f, -0.00012415929813869298f, 1.5727123022079468f));

        Alt.CreateVehicle(VehicleModel.Polmav, new Position(481.111328125f, -982.0404052734375f, 41.39368438720703f),
            new Rotation(0.0025855444837361574f, 0.00002930232585640624f, 1.5821831226348877f));
        
    }
 ```
 3) Come to helicopters (frogger2)
 4) To hear explosion sound 

### Expected behaviour

Helicopters shouldn't explode

### Operating system

Windows 11

### Version

13.0-dev13 (dev)

**Comment by YannBcf** [comment]
It only happens with mlo's (even gta default mlo's like the fib tower). It happens when the map collision hasn't loaded yet, either with a fresh client start or by driving around for enough time and for a certain distance

**Comment by YannBcf** [comment]
This has been investigated and is fixed, the fix is forwarded in release/12.11 build
